PilotDataPlatform/search
# PILOT-1744: Add support for nullable template_name

## Summary

Add support for nullable template_name

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1744

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- Check that it's possible to perform search as it's described in the bug ticket
